valhalla/baldr/edgeinfo.h

```diff
@@ -208,6 +209,9 @@ class EdgeInfo {
   // The encoded shape of the edge
   const char* encoded_shape_;
 
+  // Where we optionally keep the other half of a 64bit wayid
+  const uint32_t* extended_wayid_;
```

Comment:
we could make this even cheaper by using the spare in `w0_` and `item_`. if we did that we would only have to have a 16bit value here.

Reply:
the downside would be slightly more complex logic in the setter and getter